src/fluree/db/server_settings.clj

```diff
@@ -444,38 +444,39 @@
                                    :s3 #(s3store/close s3-conn))]
     {:storage-type storage-type
      :servers      (:fdb-conn-servers settings)
-     :options      {:transactor?    is-transactor?
-                    :storage-read   storage-read
-                    :storage-exists storage-exists
-                    ;; Storage write is overwritten in the default transactor to use the RAFT group as part of the write process
-                    ;; A default can be used here for
-                    :storage-write  storage-write
-                    :storage-rename storage-rename
-                    :storage-list   storage-list
-
-                    :req-chan       (async/chan)            ;; create our own request channel so we can monitor it if in 'dev' mode
-                    ;:sub-chan            nil
-                    ;:object-cache        nil
-                    ;:object-cache-size   nil
-                    :memory         (some-> settings :fdb-memory-cache env-bytes)
-                    :close-fn       close-fn
-                    :serializer     serializer
-
-                    ;; ledger-specific settings
-                    :tx-private-key (get-or-generate-tx-private-key settings)
-                    ;; meta is a map of settings that are implementation-specific, i.e.
-                    ;; a transactor needs novelty-min and novelty-max, a web browser connection might need some different info
-                    :meta           {:novelty-min       (-> settings :fdb-memory-reindex env-bytes)
-                                     :novelty-max       (-> settings :fdb-memory-reindex-max env-bytes)
-                                     :dev?              dev?
-                                     :password-auth     (password-feature-settings settings)
-                                     :open-api          (-> settings :fdb-api-open env-boolean)
-                                     :file-storage-path (when (= :file storage-type)
-                                                          file-ledger-storage-path)
-                                     :s3-storage        (when (= :s3 storage-type)
-                                                          {:bucket (:bucket s3-conn)
-                                                           :prefix s3-ledger-storage-prefix})
-                                     :encryption-secret encryption-key}}}))
+     :options      (cond-> {:transactor? is-transactor?
+                            :tx-private-key (get-or-generate-tx-private-key settings)}
+                     is-transactor?
+
+                     (assoc :storage-read storage-read
+                            :storage-exists storage-exists
+                            ;; Storage write is overwritten in the default transactor to use the RAFT group as part of the write process
+                            ;; A default can be used here for
```

Comment:
"can be used for" ... "now"? I realize you are just changing the indentation of the existing comment, though. :)